bubblewrap.c

```diff
@@ -951,7 +953,8 @@ parse_args (int *argcp,
             die ("--file-label not supported on this system");
 
           opt_file_label = argv[1];
-          label_create_file (opt_file_label);
+          if (label_create_file (opt_file_label))
```

Comment:
Or alternatively, givenl the duplication, maybe just fold `label_support()` and `label_valid()` together like: `die_unless_label_valid()` ?